.buildkite/bench-latency.sh

```diff
@@ -3,15 +3,21 @@
 
 set -euo pipefail
 
-bench_name=bench-latency
+bench_name_jormungandr=bench-latency-jormungandr
+bench_name_byron=bench-latency-byron
 
 echo "--- Build"
-nix-build -A benchmarks.cardano-wallet-jormungandr.latency -o $bench_name
-
-echo "+++ Run benchmark"
+nix-build -A benchmarks.cardano-wallet-jormungandr.latency -o $bench_name_jormungandr
+nix-build -A benchmarks.cardano-wallet-byron.latency -o $bench_name_byron
 
 # Note: the tracing will not work if program output is piped
 # to another process (e.g. "tee").
 # It says "Error: Switchboard's queue full, dropping log items!"
 
-./$bench_name/bin/latency
+echo "+++ Run benchmark - byron"
+
+./$bench_name_byron/bin/latency
```

Comment:
@rvl I have https://buildkite.com/input-output-hk/cardano-wallet-nightly/builds/456#005a446c-4f45-495e-ab19-896611546d7d/391-397
so probably will need to insert something here ?

Reply:
Yeah, perhaps `cd lib/byron`.

Reply:
solved in the last commit (it was about `nix/haskell.nix`) https://github.com/input-output-hk/cardano-wallet/pull/1530/commits/2648372f2bf6f7e68373d58e5349023727ab65e4

and it seems to be working (maybe it is going to be timeout) : https://buildkite.com/input-output-hk/cardano-wallet-nightly/builds/458

Reply:
oh, it works ^^^